lib/connector.js

```diff
@@ -218,6 +218,45 @@ Connector.prototype.id = function(model, prop) {
   return p && p.id;
 };
 
+/**
+ * Return the database mapping name of the property of the model if it exists.
+ * Otherwise return the property name.
+ * @param {String} model The target model name
+ * @param {String} prop The property name
+ */
+Connector.prototype.propertyMapping = function(model, property) {
+  const prop = this.getPropertyDefinition(model, property);
+  let mappingName;
+  if (prop && prop[this.name]) {
+    mappingName = prop[this.name].column || prop[this.name].columnName;
+    if (mappingName) {
+      // Explicit column name, return as-is
+      return mappingName;
+    }
+  }
+
+  // Check if name attribute provided for column name
+  if (prop && prop.name) {
+    return prop.name;
+  }
+  mappingName = property;
+  if (typeof this.dbName === 'function') {
+    mappingName = this.dbName(mappingName);
+  }
+  return mappingName;
+};
+
+/**
+ * Return the database mapping name of the id property of the model if it exists.
+ * Otherwise return the name of the id property.
+ * @param {String} model The target model name
+ * @param {String} prop The property name
+ */
```

Comment:
for the return type, similar comment as https://github.com/strongloop/loopback-connector/pull/173#discussion_r401153678 :)
and similar suggestion for the function name as https://github.com/strongloop/loopback-connector/pull/173/files#r401164434 :)